packages/core/src/plugins/mermaid.ts

```diff
@@ -0,0 +1,38 @@
+import cheerio from 'cheerio';
+import { PluginContext, FrontMatter } from './Plugin';
+
+const DEFAULT_CDN_ADDRESS = 'https://unpkg.com/mermaid@10/dist/mermaid.esm.min.mjs';
+
+function genScript(address: string) {
+  return `<script type="module">
+    import mermaid from '${address || DEFAULT_CDN_ADDRESS}';
+    document.addEventListener('DOMContentLoaded', () => {
+      Vue.directive('mermaid', {
+        inserted: function(el) {
+          mermaid.run({
+            nodes: [el]
+          });
+        }
+      });
+    });
+  </script>`;
+}
+
+export = {
+  tagConfig: {
+    mermaid: {
+      isSpecial: true,
+    },
+  },
+  getScripts: (pluginContext: PluginContext) => [genScript(pluginContext.address)],
```

Comment:
It is currently not working on the deployed website.
I think it is due to pluginContext in site.json not being pushed.